x-pack/platform/packages/shared/onechat/onechat-genai-utils/tools/search/inner_tools.ts

```diff
@@ -111,8 +112,13 @@ export const createNaturalLanguageSearchTool = ({
               },
             },
             {
+              toolResultId: getToolResultId(),
```

Comment:
@pgayvallet I'm adding this "manually" to each "tabularData" type. Is there a centralized place where I should add the id instead?

Reply:
Yeah I'm not sure. 

We could automatically add them here: https://github.com/elastic/kibana/blob/8a0d5d7692248b3e4fa7ba8c251786fc249aef47/x-pack/platform/plugins/shared/onechat/server/services/runner/run_tool.ts#L39-L63

However, this will be added very late for the "internal" tools we're using in `.search`.

Let's keep it that way for now until I have a better idea.